Dockerfile

```diff
@@ -1,2 +1,15 @@
-FROM golang:onbuild
+FROM alpine:latest
```

Comment:
@hongchaodeng needs linux based OS for Kubernetes test. `golang:onbuild` just copies the binary to the container image.